docs/src/pages/quasar-cli/cli-documentation/handling-webpack.md

```diff
@@ -57,6 +57,38 @@ The method `chainWebpack()` supplies a [webpack-chain](https://github.com/neutri
 The two examples above are equivalent. Do NOT use both methods to tamper for the same thing!
 :::
 
+## Examples
+
+### CopyWebpackPlugin
+
+One of the most asked questions is how to copy independent files to the destination folder (ie: `dist/spa`).
+
+Below, uses the `CopyWebpackPluging` webpack plugin using `extendWebpack` to copy `robots.txt` to the build destination.
+
+First, add `copywebpackplugin` to your `package.json`.
+
+```js
+yarn add -D copy-webpack-plugin
+```
+
+At the top of `quasar.conf.js` add the following:
+
+```js
+const CopyWebpackPlugin = require('copy-webpack-plugin')
+```
+
+In `quasar.conf.js` look for the `build` section. Add this:
+
+```js
+  extendWebpack (cfg) {
+    cfg.plugins.push(new CopyWebpackPlugin([{ from: './robots.txt', to: '' }]))
+  }
+```
+
+If you already have `extendWebpack (cfg)`, then just integrate the `cfg.plugins.push` part appropriately.
+
+Also note, the `to` parameter is empty. It already knows your destinatin path via webpack. This is the `root`. If you add a path, it will be based on the destination. You may have also noticed that the parameter to `CopyWebpackPlugin` is an array of objects. You can add multiple files here, one object for each file.
```

Comment:
```suggestion
Also note, the `to` parameter is empty. It already knows your destination path via Webpack. This is the `root`. If you add a path, it will be based on the destination. You may have also noticed that the parameter to `CopyWebpackPlugin` is an array of objects. You can add multiple files here, one object for each file.
```